### ISSUE TITLE
[BUG] Fails to compile assets

### ISSUE THREAD

**Issue by Supernova3339** [tiredofit/docker-discourse#1]
### Summary

Fails to compile assets
It's been 2 hours, and it still hasn't finished. I think something must've gone wrong.

### Steps to reproduce

1. Execute the image

### What is the expected *correct* behavior?

Should compile assets

### Environment

- Image version / tag:  tiredofit/discourse:3.1.0
- Host OS: Ubuntu 22

<details>
<summary>Any logs | docker-compose.yml</summary>
</details>

**Comment by tiredofit** [comment]
You can try `SHOW_OUTPUT=TRUE` that might give some insight.

**Comment by Supernova3339** [comment]
I'll give it a shot, thanks!

**Comment by Supernova3339** [comment]
Doesn't even show logs now.

**Comment by tiredofit** [comment]
Also you can try DEBUG_MODE=TRUE

**Comment by Supernova3339** [comment]
Does the same thing. If may ask, does the container work with docker swarm?

**Comment by tiredofit** [comment]
Not sure. Theoretically it should - but everyone's environment could be different in terms of things like volumes, network overlays, etc.

**Comment by Supernova3339** [comment]
Had no luck sadly. Does the image work for you?

**Comment by tiredofit** [comment]
Yes, running and handling duties for a couple hundred users without issue..

**Comment by Supernova3339** [comment]
I'll try building it from scratch, and I'll let you know how it goes.
Been wanting to get it added to easypanel <https://easypanel.io> for awhile
now. Already got freescout added, your images are awesome :)

**Comment by tiredofit** [comment]
I don't actually - It was a real pain for me to build this a couple years ago and I spent a day or so in December getting it modernized and its been running quite well on my infrastructure. I'm not sure what else to say unless I had access to the environment in question although that would require some sort of private support and time to schedule that work in as I'm quite booked up these days. I just fired up a sample install (based on the example) and it worked without issue on a amd64 system and was quite speedy - I'm really not sure what is happening on that end..

**Comment by monim67** [comment]
Having the same issue, it just freezes (may be doing nothing at all)

```
discourse-app  | 2023-06-30.21:00:59 [NOTICE] ** [discourse] Enabling Plugin: Presence
discourse-app  | 2023-06-30.21:00:59 [NOTICE] ** [discourse] Enabling Plugin: Style Guide
discourse-app  | 2023-06-30.21:01:00 [INFO] ** [discourse] Migrating Database - This may take a bit..
discourse-app  | /assets/functions/00-container: line 960:  2656 Aborted                 "$@" > /dev/null 2>&1
discourse-app  | 2023-06-30.21:01:14 [INFO] ** [discourse] Precompiling Assets - This may take a bit..
```

With DEBUG_MODE=true this is where it stops forever.

```
discourse-app  | + print_info 'Migrating Database - This may take a bit..'
discourse-app  | + output_off
discourse-app  | + '[' true = true ']'
discourse-app  | + set +x
discourse-app  | 2023-06-30.21:25:44 [INFO] /etc/cont-init.d/20-discourse ** [discourse] Migrating Database - This may take a bit..
discourse-app  | + cd /app
discourse-app  | + silent bundle exec rake db:migrate
discourse-app  | + '[' TRUE = true ']'
discourse-app  | + '[' true = true ']'
discourse-app  | + bundle exec rake db:migrate
```
